src/App.tsx

```diff
@@ -15,71 +16,122 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type BtnProps = {
+  dataCy: string,
+  content: string,
+  classes?: string[],
+  handler: () => void
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const GoodsButton: React.FC<BtnProps> = (
+  {
+    dataCy,
+    content,
+    handler,
+    classes = [],
+  },
+) => (
+  <button
+    data-cy={dataCy}
+    type="button"
+    className={classNames(...classes)}
+    onClick={() => handler()}
+  >
+    {content}
+  </button>
+);
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+GoodsButton.defaultProps = {
+  classes: ['button'],
+};
```

Comment:
better do it during destructiring, it's more clear this way, I almost missed your defaultProps